### DISCUSSION TITLE
Can the 'Invasion Percolation' simulate the evaporation process?

### DISCUSSION THREAD

**Discussion by mourendong** [PMEAL/OpenPNM#2200] [Q&A]
Hi, everyone,
I try to simulate the evaporation process by the OpenPNM.
The 'Invasion Percolation' can show the invasion process.
Can the 'Invasion Percolation' simulate the evaporation process? And how to consider the airflow rate and water flow rate?

**Comment by jgostick** [comment]
Not directly.  You'll have to manually couple the various algorithms directly.  We did [a paper on this](https://doi.org/10.1007/s10800-017-1126-6) a few years ago, and it was quite a pain.
